build.sbt

```diff
@@ -1,13 +1,12 @@
-lazy val `sbt-js-engine` = project in file(".")
+lazy val `sbt-js-engine` = (project in file(".")).enablePlugins(SbtWebBase)
 
 description := "sbt js engine plugin"
 
 libraryDependencies ++= Seq(
-  "com.typesafe" %% "jse" % "1.2.4",
-  "com.typesafe" %% "npm" % "1.2.1",
```

Comment:
`npm` comes with `jse` already, no need to add it again:
https://github.com/typesafehub/npm/blob/1.2.2/build.sbt#L14